sympy/polys/domains/tests/test_domains.py

```diff
@@ -579,8 +579,8 @@ def check_element(e1, e2, K1, K2, K3):
 
     def check_domains(K1, K2):
         K3 = K1.unify(K2)
-        check_element(K3.convert_from(K1.one,  K1), K3.one , K1, K2, K3)
-        check_element(K3.convert_from(K2.one,  K2), K3.one , K1, K2, K3)
+        check_element(K3.convert_from( K1.one, K1),  K3.one, K1, K2, K3)
+        check_element(K3.convert_from( K2.one, K2),  K3.one, K1, K2, K3)
```

Comment:
Not sure about this one

Reply:
Well, it seemed like the idea was to align the commas, but one could always argue exactly how it should be done. Now it is at least consistent (which is wasn't earlier).